### ISSUE TITLE
Symfony 2.7 update

### ISSUE THREAD

**Issue by danrot** [sulu-io/sulu#1255]

**Comment by chirimoya** [comment]
ping @wachterjohannes

**Comment by chirimoya** [comment]
this should make it into 1.1 /cc @danrot @wachterjohannes @berndhep

**Comment by danrot** [comment]
The PHPCR-ODM library still has version 1.3 not released, with the fix for the deprecation notice. If we merge that we are probably having a lot of deprecation notices...

**Comment by cbastienbaron** [comment]
hi @danrot ,

PHPCR-ODM has been released https://github.com/doctrine/phpcr-odm/releases/tag/1.3.0 
Can we bump symfony's dep version to 2.7 ? :) it's a blocking point sf 2.6.* for us :+1:

**Comment by danrot** [comment]
@cbastienbaron Yeah, I've seen that. We said that we will make a direct upgrade to 2.8. Would that also be ok for you?

**Comment by cbastienbaron** [comment]
that would be perfect :+1:  :+1:  :+1:

**Comment by danrot** [comment]
Ok, that will be done on develop quite soon. 1.2 will then be released with Symfony 2.8. In the mean time you could simply change the dependency to 2.7, I think @tomsvogel uses it like that.

**Comment by cbastienbaron** [comment]
seems to have some test failed with 2.8 

```
2) Sulu\Bundle\SnippetBundle\Controller\SnippetControllerTest::testCopyLocale
Symfony\Component\DependencyInjection\Exception\RuntimeException: You have requested a synthetic service ("kernel"). The DIC does not know how to construct this service.
```
